docs/config.rst

```diff
@@ -435,11 +436,21 @@ The algorithm used by trust manager factory for SSL connections. Default value i
   * Importance: low
 
 Configuration Options for SASL Authentication between REST Proxy and Apache Kafka Brokers
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-----------------------------------------------------------------------------------------
+
+Kafka SASL configurations are described `here <http://docs.confluent.io/3.0.0/kafka/sasl.html>`_
 
 Note that all the SASL configurations (for REST Proxy to Broker communication) are prefixed with "client". If you want the configuration to apply just to consumers or just to producers, you can replace the prefix with "consumer" or "producer" respectively.
 
-In addition to these configurations, make sure ``bootstrap.servers`` configuration is set with SASL_PLAINTEXT://host:port (or SASL_SSL) end-points, or you'll accidentally open an SASL connection to a non-SASL port.
+In addition to these configurations, make sure ``bootstrap.servers`` configuration is set with SASL_PLAINTEXT://host:port (or SASL_SSL://host:port) end-points, or you'll accidentally open an SASL connection to a non-SASL port.
+
+Pass the name of the JAAS file as a JVM parameter to each Kafka broker, schema registry, and REST Proxy. For example:
+
+ -Djava.security.auth.login.config=/etc/kafka/kafka_server_jaas.conf
```

Comment:
Where do we pass it? Maybe give the entire command as an example? 

(same below)